packages/aws-cdk-lib/aws-sns/lib/delivery-policy.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Algorithms which can be used by SNS to calculate the delays associated with all of the retry attempts between the first and last retries in the backoff phase.
+ */
+export enum BackoffFunction {
+  /** Arithmetic.
+   */
+  ARITHMETIC = 'ARITHMETIC',
+  /** Geometric.
+   */
+  EXPONENTIAL = 'EXPONENTIAL',
+  /** Linear.
+   */
+  GEOMETRIC = 'GEOMETRIC',
+  /** Linear.
+   */
+  LINEAR = 'LINEAR',
+}
+
+/**
+ * Options for customising AWS SNS HTTP/S delivery throttling.
+ */
+export interface ThrottlePolicy {
+  /** The maximum number of deliveries per second, per subscription.
+   * @default - no throttling
+   */
+  readonly maxReceivesPerSecond?: number;
+}
+
+/**
+ * Options for customising aspects of the content sent in AWS SNS HTTP/S requests.
+ */
+export interface RequestPolicy {
+  /** The content type of the notification being sent to HTTP/S endpoints.
+   * @default - text/plain; charset=UTF-8
+   */
+  readonly headerContentType?: string;
+}
+
+/**
+ * Options for customising the retry policy of the delivery of SNS messages to HTTP/S endpoints.
+ */
+export interface HealthyRetryPolicy {
+  /** The minimum delay for a retry (in seconds).  Must be at least 1 and not exceed `maxDelayTarget`.
+   * @default 20
+   */
+  readonly minDelayTarget: number;
```

Comment:
Only, this and the next two appear to be mandatory if `healthyRetryPolicy` is specified, thought I can't find any specific doc for this.

Reply:
Would it be correct to assume that if these are not set, it will result in an error during deployment? If so, I am curious when the default values would be set.

Reply:
If I recall correctly from testing, it's required if and only if `healthyRetryPolicy` itself is specified, and so the default values would be set in the case `healthyRetryPolicy` is not specified (perhaps it's confusing to mention a default in the doc :thinking:)

Reply:
How about making these two props optional and explicitly setting the default values documented within the L2 construct? These properties may be required in CloudFormation, but I felt there is no need to make them mandatory in the L2 construct.

```ts
  readonly minDelayTarget?: Duration;
  readonly maxDelayTarget?: Duration;
```

```ts
const minDelayTarget = healthyRetryPolicy.minDelayTarget ?? Duration.seconds(20);
const maxDelayTarget = healthyRetryPolicy.maxDelayTarget ?? Duration.seconds(20);
```